app/services/qa/linked_data/mapper/context_mapper_service.rb

```diff
@@ -0,0 +1,46 @@
+# Provide service for mapping predicates to object values.
+module Qa
+  module LinkedData
+    module Mapper
+      class ContextMapperService
+        class_attribute :graph_service
+        self.graph_service = Qa::LinkedData::GraphService
+
+        class << self
+          # Extract predicates specified in the predicate_map from the graph and return as a value map for a single subject URI.
+          # @param graph [RDF::Graph] the graph from which to extract result values
+          # @param context_map [Qa::LinkedData::Config::ContextMap] defines properties to extract from the graph to provide additional context
+          # @param subject_uri [RDF::URI] the subject within the graph for which the values are being extracted
+          # @return [<Hash<Symbol><Array<Object>>] mapped context values and information with hash of map key = array of object values for predicates identified in predicate_map.
+          # @example value map for a single result
+          #   {:uri=>[#<RDF::URI:0x3fcff54a829c URI:http://id.loc.gov/authorities/names/n2010043281>],
```

Comment:
This should start with a `[`

Reply:
I also thing if I am reading the code correctly the example should really look like below.  I do not see id, label, altlabel being added in construct context.
```
[{"group" => "group label,
   "property" => "property label",
   "values" => ["value 1","value 2"],
  "selectable" => true,
  "drillable" => false}]
```

Reply:
That is a copy paste error.  I'll update the example.